evergreen/install_baas.sh

```diff
@@ -340,7 +365,7 @@ git config --global url."git@github.com:".insteadOf "https://github.com/"
 
 # If a baas branch or commit version was not provided, retrieve the latest release version
 if [[ -z "${BAAS_VERSION}" ]]; then
-    BAAS_VERSION=$(${CURL} -LsS "https://realm.mongodb.com/api/private/v1.0/version" | jq -r '.backend.git_hash')
+    get_var_from_file BAAS_VERSION "${BASE_PATH}/../dependencies.list"
```

Comment:
You will also need to transfer the dependencies.list file to the remote baas host in evergreen/setup_baas_host_local.sh for running baas on a remote host.

Reply:
is it related to this error? `./evergreen/install_baas.sh: line 223: /data/baas-remote/evergreen/../dependencies.list: No such file or directory`